src/pytest_dbt_core/fixtures.py

```diff
@@ -58,6 +59,12 @@ def config(request: SubRequest) -> RuntimeConfig:
     RuntimeConfig
         The runtime config.
     """
+
+    # only override profiles_dir if its set, otherwise keep DBT behavior
```

Comment:
I prefer to keep "why" comments only. This comment describes what I can read from the code

```suggestion
```

Reply:
removed https://github.com/godatadriven/pytest-dbt-core/pull/25/commits/9c04c9242202d3573ee12a37a566f7a3bc60fcea